metadata-ingestion/tests/unit/sql_queries/test_sql_queries_enhancements.py

```diff
@@ -0,0 +1,766 @@
+import re
+from unittest.mock import Mock, patch
+
+import pytest
+
+from datahub.ingestion.source.sql_queries import (
+    SqlQueriesSource,
+    SqlQueriesSourceConfig,
+)
+
+
+class TestPerformanceOptimizations:
```

Comment:
```suggestion
class TestPerformanceConfigOptimizations:
```

just to reduce expectations 😅